desc/experimental/_terpsichore.py

```diff
@@ -0,0 +1,872 @@
+import functools
+import multiprocessing as mp
+import os
+import shutil
+import subprocess
+import time
+
+import numpy as np
+from netCDF4 import Dataset, stringtochar
+
+from desc.backend import execute_on_cpu, jnp
+from desc.basis import DoubleFourierSeries
+from desc.compat import flip_theta
+from desc.compute.utils import get_transforms
+from desc.grid import LinearGrid
+from desc.objectives import ExternalObjective
+from desc.objectives.objective_funs import collect_docs
+from desc.transform import Transform
+from desc.utils import errorif, warnif
+from desc.vmec_utils import ptolemy_identity_rev, zernike_to_fourier
+
+
+@execute_on_cpu
+def terpsichore(
+    eq,
+    processes=1,
+    path="",
+    exec="",
+    scalar=True,
+    mode_family=-1,
+    surfs=101,
+    lssl=1000,
+    lssd=1000,
+    M_max=8,
+    N_min=-4,
+    N_max=4,
+    M_booz_max=19,
+    N_booz_max=18,
+    awall=2.0,
+    deltaJp=1e-2,
+    modelk=1,
+    nev=1,
+    al0=-5e-1,
+    sleep_time=1,
+    stop_time=60,
```

Comment:
what is sleep_time and stop_time in? seconds?

Reply:
Yes, seconds. It's documented in the `TERPSICHORE` objective function docstring (that is the part that is visible to the user, this is just an internal intermediate function).